panther_manager/include/panther_manager/manager_bt_node.hpp

```diff
@@ -45,13 +58,23 @@ class ManagerNode
   BT::Tree safety_tree_;
   BT::Tree shutdown_tree_;
 
-  void setup_behavior_tree(BT::Tree & tree, BT::NodeConfig & config, const std::string tree_name);
+  std::unique_ptr<MovingAverage<double>> battery_temp_ma_;
+  std::unique_ptr<MovingAverage<double>> battery_percent_ma_;
+  std::unique_ptr<MovingAverage<double>> cpu_temp_ma_;
+  std::unique_ptr<MovingAverage<double>> front_driver_temp_ma_;
+  std::unique_ptr<MovingAverage<double>> rear_driver_temp_ma_;
+
+  void battery_cb(const sensor_msgs::BatteryState::ConstPtr & battery);
+  void driver_state_cb(const panther_msgs::DriverState::ConstPtr & driver_state);
   void e_stop_cb(const std_msgs::Bool::ConstPtr & e_stop);
   void io_state_cb(const panther_msgs::IOState::ConstPtr & io_state);
+  void system_status_cb(const panther_msgs::SystemStatus::ConstPtr & system_status);
   void safety_tree_timer_cb();
   void lights_tree_timer_cb();
+  void shutdown_robot(const std::string & message);
+  BT::NodeConfig create_bt_config(const std::map<std::string, std::any> bb_values = {}) const;
```

Comment:
```suggestion
  BT::NodeConfig create_bt_config(const std::map<std::string, std::any> & bb_values = {}) const;
```
That was actually copying the whole map before delegating this function